tps-client/src/lib.rs

```diff
@@ -140,6 +141,7 @@ pub trait TpsClient {
     ) -> TpsClientResult<UiConfirmedBlock>;
 }
 
+#[cfg(feature = "bank-client")]
```

Comment:
sounds like a good idea to hide this under the feature. One thing is that I'm not quite sure it is used anywhere, maybe be in integration tests (?) in this case would be enough to have development feature